data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,472 @@ mission "Unfettered returning home"
+			`	"Furthermore, we can understand some may die in glorious battle, and to the victor go the spoils, so looting is allowed. Capturing, however, is off limits. Do so, and all the Unfettered will be against you."`
+			# This is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles or so.
+			`	He suddenly looks at you intensively, straight in the eyes. "We sometimes bend the rules for our friends, but that is not the case with you," he says with a sudden change of tone before leaving.`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	Mera comes back, looking a bit impatient. "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships. The engineer pulls you in close once they are far enough away. "I want this done by the end of the day, so do not try my patience. You are lucky I offered this deal in the first place, as they were planning to take the relic by force. Now, quick, get to your ship!" You see the Unfettered have already begun their launching procedures, so you prepare for takeoff as soon as you can.`
+			`	(Remember to park your other ships and make sure your flagship is the Anomalocaris.)`
+				accept
+			label headingout
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				set "unfettered: fled with the relic"
+			`	Seeing as they are unlikely to cooperate, you quickly head to your ship and launch before they have time to react, but shortly after they begin to follow you.`
+				launch
+	on enter
+		conversation
+			branch fleeing
+				has "unfettered: fled with the relic"
+			branch "too many ships"
+				"total ships" > 1
+			`As you leave the planet's atmosphere, you see the challengers already waiting for you in the system. They hail you, "Luck has nothing to do with it. Show us what you got, monkey! Show us that you control your weapons well, as any warriors should. Prove to us that you have the strength to disable our ship, yet still have the restraint not to destroy them."`
+				accept
+			label fleeing
+			action
+				fail
+			`The Unfettered don't seem too pleased to see you leaving with their prize. The challengers are already waiting for you in orbit, and hail you. "Where do you think you're going, monkey?"`
+				decline
+			label "too many ships"
+			`As you take to join your fleet in orbit, You receive a hail from the Unfettered, "If you can't even count how many ships you have you're not worthy of our time, monkey!"`
+			action
+				fail
+	on disabled
+		fail
+		conversation
+			`The enemy ships hail you. "Thought you could get away from us so easily? You better bribe us and then land on our worlds quick, before we value blowing you up more than preserving that ship!"`
```

Comment:
```suggestion
			`The enemy ships hail you. "Thought you could get away from us so easily? You better bribe us and then land on our worlds quick, before we decide that we value blowing you up more than preserving that ship!"`
```

Reply:
I feel like this takes away a bit of the Hai's character; it makes them sound a bit too thorough in their speech. The original is fine.

Reply:
It doesn't sound grammatically correct, but I suppose they don't speak Basic as well as the player.

Reply:
It is grammatically correct. It's saying they value blowing something up more. Stating that you value one thing over the other, regardless of it its an action or object, is grammatically correct. Your option is too, but it's changing the value from the action itself to the decision of the action, and is slightly more wordy.

Reply:
_sound_. I suppose though.